sensor_msgs/Image image
float32[] range
